sdk/communication/communication-call-automation/README.md

```diff
@@ -116,17 +122,17 @@ See below for example: where you are making a call with CreateCall, and wait for
 
 ```ts snippet:ReadmeSampleEventProcessorExample
 import { DefaultAzureCredential } from "@azure/identity";
-import { CallAutomationClient } from "@azure/communication-call-automation";
+import { CallAutomationClient, CallInvite } from "@azure/communication-call-automation";
 
 // Your unique Azure Communication service endpoint
 const credential = new DefaultAzureCredential();
 const endpointUrl = "<ENDPOINT>";
 const callAutomationClient = new CallAutomationClient(endpointUrl, credential);
 
 // send out the invitation, creating call
-const callInvite = new CallInvite(target);
+const callInvite: CallInvite = { targetParticipant: { communicationUserId: "8:acs:..." } };
```

Comment:
In other places we've removed the type annotations, like `: CallInvite`, should we remove it here too? Not sure if we're aiming for plain JavaScript in the snippets.

Reply:
Fixed.